src/main/java/com/puppycrawl/tools/checkstyle/Main.java

```diff
@@ -709,6 +740,11 @@ private static Options buildOptions() {
         options.addOption(OPTION_C_NAME, true, "Sets the check configuration file to use.");
         options.addOption(OPTION_O_NAME, true, "Sets the output file. Defaults to stdout");
         options.addOption(OPTION_P_NAME, true, "Loads the properties file");
+        options.addOption(OPTION_S_NAME, true,
+                "Print xpath suppressions at the file's line and column position");
+        options.addOption(OPTION_TAB_WIDTH_NAME, true,
+                "Sets the length of the tab character. Used only with \"-s\" option. "
+                        + "Default value is 8");
```

Comment:
Default open should be an argument with value being passed in from DEFAULT_TAB_WIDTH .
No one will remember to update documentation is default is changed.
This is how option `f` is done.

Reply:
Done